samples/cpp/facedetect.cpp

```diff
@@ -122,36 +114,27 @@ int main( int argc, const char** argv )
         if(image.empty()) cout << "Couldn't read ../data/lena.jpg" << endl;
     }
 
-    cvNamedWindow( "result", 1 );
+    namedWindow( "result", 1 );
```

Comment:
could you show the proper place to put this line and the comment or what is your opinion about removing complately?